test/passing/tests/doc.mld.ref

```diff
@@ -0,0 +1,164 @@
+{0 Parent/Child Specification}
+
+This parent/child specification allows more flexible output support, e.g.,
+per library documentation. See {{:https://v3.ocaml.org/packages}
+v3.ocaml.org/packages}.
```

Comment:
Not really related to this PR but unanswered: Formatting of links could be improved.

Reply:
done in #2152